docker-helper

```diff
@@ -1,22 +1,22 @@
 #!/bin/bash
 
-set -euo pipefail
+set -eo pipefail
```

Comment:
why?

Reply:
I couldn't leave out arguments $4, $5 and $6 because set -u treats unset parameters as errors. My main motivation for this refactor is leaving out $4-6, so I thought the -u was unneeded.